src/main/java/de/retest/recheck/ui/descriptors/Element.java

```diff
@@ -213,6 +213,21 @@ public Attributes getAttributes() {
 		return attributes;
 	}
 
+	/**
+	 * Returns the value of the given attribute, whether this is an identifying attribute or not.
+	 *
+	 * @return The {@link java.io.Serializable} value of the attribute.
+	 */
+	public Object getAttributeValue( final String attributeName ) {
```

Comment:
(Optional) suggestion to avoid repeated calls to getters:

```java
public Object getAttributeValue( final String attributeName ) {
	Object identifyingAttribute = getIdentifyingAttributes().get( attributeName );
	if ( identifyingAttribute != null ) {
		return identifyingAttribute.getValue();
	}

	Object ordinaryAttribute = getAttributes().get( attributeName );
	if ( ordinaryAttribute != null ) {
		return ordinaryAttribute.getValue(); // Doesn't this require getValue() too?
	}

	return null;
}
```